vhost-device-gpu/Cargo.toml

```diff
@@ -14,9 +14,10 @@ edition = "2021"
 resolver = "2"
 
 [features]
-default = ["gfxstream"]
+default = ["backend-virgl", "backend-gfxstream"]
 xen = ["vm-memory/xen", "vhost/xen", "vhost-user-backend/xen"]
-gfxstream = ["rutabaga_gfx/gfxstream"]
+backend-gfxstream = ["rutabaga_gfx/gfxstream"]
```

Comment:
This should be changed in a separate commit IMO, explaining the reaason

Reply:
I seperated the commits, but this hunk I put them together, let me know if it make sense?